app/main.py

```diff
@@ -21,14 +21,16 @@ def __iadd__(self, other):
         return self
 
     def __mul__(self, other):
-        if isinstance(other, Distance):
-            return Distance(self.km * other.km)
-        return Distance(self.km * other)
+        if isinstance(other, (int, float)):
+            return Distance(round(self.km * other, 2))
+        else:
+            raise TypeError(f"{type(other)} can`t be multiply on {type(self.km)}")
```

Comment:
I can't multiply class object by class object. Write it in the condition, and remove this TypeError line.

Reply:
https://github.com/mate-academy/py-distance-class/pull/8#discussion_r787886731 I followed the instructions of the mentor, added an TypeError because, according to the condition from the example, these operations are available only for (int, float) types. Now i little bit confused what i need to fix.

Reply:
"other can't be an instance of Distance only a number (float or int)" i check it , if other is not int or float i return error becouse the meaning of the Distance class is to perform operations with distance, not area, also I cannot multiply the distance by a string or another class, so I will check for valid types otherwise I return an exception. I understand that there is a number in another class, but it is written in a variable that denotes kilometers, and a kilometer per kilometer is an area